snaps_tests/demos_tests/test_git.py

```diff
@@ -0,0 +1,31 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2016 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import snaps_tests
+
+
+class GitTestCase(snaps_tests.SnapsTestCase):
+
+    snap_content_dir = 'git'
+
+    def test_git(self):
+        self.build_snap(self.snap_content_dir)
+        self.install_snap(self.snap_content_dir, 'git', '0')
+        self.snappy_testbed.run_command(['sudo', 'snap', 'connect',
+                                         'git:home', 'ubuntu-core:home'])
+        self.assertTrue(
+            self.snappy_testbed.run_command(['/snap/bin/git',
+                                             '--version']).startswith('git'))
```

Comment:
Thank for adding the test!
But I was thinking that you could do something like:
self.snappy_testbed.run_command(['mkdir', '-p', '$HOME/tmp/foo'])
self.snappy_testbed.run_command(['/snap/bin/git', 'init', '$HOME/tmp/foo'])

This might need some more work on the run helpers. I'm around to help.